ui/window.go

```diff
@@ -265,7 +265,12 @@ func (win *Window) drawWindowUI(gtx C) {
 // callers should not re-refresh the display.
 func (win *Window) changePage(page load.Page, keepBackStack bool) {
 	if win.currentPage != nil && keepBackStack {
-		win.currentPage.OnNavigatedFrom()
+		// prevent page from resetting main page OnNavigatedFrom data as restore page
+		// use the changePage() as using the popFragement() or changeFragement()
+		// renders the restore ui on a much smaller layout and distorts the display
```

Comment:
```
When navigating to the Restore page from the MainPage, don't clear the current
page (main page) data using the `currentPage.OnNavigatedFrom()` method. The
Restore page will pop back to the main page when done and we'll need the data
untouched. The Restore page is being opened using changePage() instead of
changeFragment() because the latter renders the restore page UI on a much
smaller area and distorts the display. Also, the Main page's data will NOT be reset
when the Restore page is exited via popPage().
```